packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -3358,7 +3372,15 @@ export default function InputGalleryCreateForm(props) {
         isReadOnly={false}
         type=\\"datetime-local\\"
         onChange={(e) => {
-          let value = Number(new Date(e.target.value));
+          const date = new Date(e.target.value);
+          if (!(date instanceof Date && !isNaN(date))) {
```

Comment:
Actually, if the field isn't required, empty actually should be a valid value for the input . I think that's the case for all inputs we are casting.

Reply:
in regards to datetime-local fields the onChange is not executed for incomplete dates or empty dates